src/main/java/com/world/demo/worldmap/controller/CountryController.java

```diff
@@ -44,7 +39,7 @@ public ResponseEntity getCountryByCode(@PathVariable String code) {
             }
             return new ResponseEntity<>("INVALID_COUNTRY_CODE", HttpStatus.INTERNAL_SERVER_ERROR);
```

Comment:
Do not use an Internal Server Error here. If a country was found just return the Country object, if not you could throw something like a custom CountryNotFoundException wich is mapped to status 404 by `@ResponseStatus`.